### ISSUE TITLE
VPAT Reader: missing aria labels and roles

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#4220]
- vpat 60: Sidebar search close button missing name and incorrect role
    -  On mac with voiceover enabled, focus the "Search annotations" input field, type something, click right arrow, "X" icon is focused and it says "image" instead of a descriptive label.
- vpat 64: Sidebar Edits Inputs For Annotations Lack Accessible Names
    - Referring to `<Editor>` for annotations such as highlight, note, text, underline
- vpat 65: Annotation Menu Buttons (`class="more"` button that open context menu) Lack Accessible Names
- vpat 69: Sidebar Tag Button Name Not Descriptive After Tag Added
    - If an annotation has no tag, `<button class="tags"` has "Add tags..." name which is announced by screen readers, and it makes it clear what the button does. But if a tag is added, screen readers will announce the tags name instead, which makes it less clear what the purpose of the button is. We may want to add an `aria-description="Click to add more tags"` or something like that.
- vpat 71: Sidebar Annotations Only Announced As "Section" By NVDA
    - I actually cannot quite reproduce this as NVDA announces the entire annotation for me ok. But JAWS doesn't and the first thing that is announced on all screen readers is the tooltip with the date and time of annotation, which is not the most descriptive. So we should maybe have a proper aria name for each annotation box.
- vpat 74: Sidebar Tag Buttons Purposes Are Unclear
    - From the vpat review: "their purpose as filters for annotations is not clearly conveyed. Tag buttons must have an accessible name or description that clarifies their functionality." Perhaps, we should indicate that it's a tag, since colored tags are announced as just "yellow" or "blue", which can also be not immediately clear that it refers to all yellow or blue annotations
 - vpat 79: Page Change Edit Inputs Lack Labels and Names
     - Annotation in the side bar > Context Menu > Edit Page Number ...  - a `class="modal-popup"` appears and the input for the page number is unlabeled
